### PR TITLE
Beatmapset hypes count

### PR DESCRIPTION
**Pull request by nanaya**
Inflicting elasticsearch to everyone =]

The docker needs updating though.

    $ artisan beatmapsets:hypessync

(hypes? hype? 😕 )

### CODE REVIEW

**Comment by nekodex** [comment]
p. sure hype is an uncountable noun /shrug

File: app/Console/Commands/BeatmapsetsHypeSyncCommand.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->info('Synchronising hype counts...');
+
+        $this->progress = $this->output->createProgressBar(Beatmapset::count());
+
+        Beatmapset::chunk(1000, function ($sets) {
```
Comment (notbakaneko) [inline app/Console/Commands/BeatmapsetsHypeSyncCommand.php]: `chunkById`

File: .travis.yml [hunk 2]
```diff
@@ -23,13 +23,22 @@ env:
...
   - nvm use 6.11
   - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.27.5
   - ./bin/db_setup.sh
   - ./build.sh
 
+  # elasticsearch setup
+  - wget "${ES_DOWNLOAD_URL}"
+  - unzip "elasticsearch-${ES_VERSION}.zip"
+  - ./"elasticsearch-${ES_VERSION}/bin/elasticsearch" &
+  - sleep 10 # wait until the server is up
```
Comment (notbakaneko) [inline .travis.yml]: what could possibly go wrong :trollface:

Comment (nanaya) [reply .travis.yml]: one of the methods they recommend 💃 

https://docs.travis-ci.com/user/database-setup/#ElasticSearch

**Pull request merged**